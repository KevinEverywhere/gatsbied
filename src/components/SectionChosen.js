import React from 'react'
import PropTypes from 'prop-types'
import ItemChooser from './ItemChooser'

const SectionChosen = props => (
  <nav>
    <ul>
      <ItemChooser item="Work" />
      <ItemChooser item="Creative" />
      <ItemChooser item="Contact" />
    </ul>
  </nav>
)

export default SectionChosen
