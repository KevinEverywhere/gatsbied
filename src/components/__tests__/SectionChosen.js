import React from "react"
import renderer from "react-test-renderer"
import SectionChosen from "../SectionChosen"

describe("SectionChosen", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<SectionChosen />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
