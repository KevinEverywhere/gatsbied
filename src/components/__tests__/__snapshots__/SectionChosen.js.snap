// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SectionChosen renders correctly 1`] = `
<nav>
  <a
    href="/"
  >
    <div
      className="smlogo"
    >
      <span
        className="icon fa-bicycle"
      />
    </div>
  </a>
  <ul>
    <li>
      <a
        href="work"
      >
        Work
      </a>
    </li>
    <li>
      <a
        href="creative"
      >
        Creative
      </a>
    </li>
    <li>
      <a
        href="contact"
      >
        Contact
      </a>
    </li>
  </ul>
</nav>
`;
